scripts/install-chef-suse.sh

```diff
@@ -291,7 +291,7 @@ function reset_crowbar()
 {
     rm -f $crowbar_install_dir/crowbar-install-failed
     rm -f $installation_steps
-    su -s /bin/sh - crowbar sh -c "cd /opt/dell/crowbar_framework && RAILS_ENV=production bin/rake db:cleanup"
+    su -s /bin/sh - crowbar sh -c "cd /opt/dell/crowbar_framework && RAILS_ENV=production bin/rake db:cleanup db:seed"
```

Comment:
Why is a db:seed needed here? db:cleanup looks like a task which basically migrate the database. Did I get something wrong?